README.md

```diff
@@ -1085,33 +1085,15 @@ ANSWERS.addComponent('FilterOptions', {
   container: '.filter-container',
   // Control type, singleoption or multioption
   control: 'singleoption',
+  // The type of filtering to apply to the options, either 'STATIC_FILTER' or 'RADIUS_FILTER'.
+  // Defaults to 'STATIC_FILTER'.
```

Comment:
I wanted to avoid documentation that basically takes a config option like chocolateChipCookie and tells you "it is a cookie made of chocolate chips" but I think what you said makes sense here

Reply:
I think it's okay to say that optionType represents the type or kind of the options if we also provide some quick details about where 'STATIC_FILTER' options (profile field values) and 'RADIUS_FILTER' options (distances) come from. If we provide the additional details about what the different kinds of options represent, you're providing the reader with additional info apart from the 'optionType is type of options' tautology. You don't need much detail here since you expand on them below.